### ISSUE TITLE
Unable to install on macOS Ventura (13.0.1)

### ISSUE THREAD

**Issue by jonathanstegall** [Automattic/vip-cli#1172]
## Expected/Desired Behavior

I'm trying to install or upgrade the cli tool. It was unable to upgrade, so I tried uninstalling node entirely and reinstalling all of my global packages. The only one that is still failing is vip.

## Actual Behavior

How does the CLI currently work.

## Steps to Reproduce the Problem

When I run `npm install -g @automattic/vip`, here's the result:
```
npm ERR! code 1
npm ERR! path /usr/local/lib/node_modules/@automattic/vip/node_modules/re2
npm ERR! command failed
npm ERR! command sh -c -- node-gyp rebuild
npm ERR! CXX(target) Release/obj.target/re2/lib/addon.o
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.1.0
npm ERR! gyp info using node@19.1.0 | darwin | x64
npm ERR! gyp info find Python using Python version 3.10.8 found at "/usr/local/opt/python@3.10/bin/python3.10"
npm ERR! gyp info spawn /usr/local/opt/python@3.10/bin/python3.10
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/usr/local/lib/node_modules/@automattic/vip/node_modules/re2/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/Users/jonathanstegall/Library/Caches/node-gyp/19.1.0',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/usr/local/lib/node_modules/npm/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/<(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/usr/local/lib/node_modules/@automattic/vip/node_modules/re2',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp info spawn make
npm ERR! gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
npm ERR! In file included from ../lib/addon.cc:1:
npm ERR! In file included from ../lib/./wrapped_re2.h:4:
npm ERR! In file included from ../../nan/nan.h:60:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/node.h:73:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8.h:24:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-array-buffer.h:12:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-local-handle.h:12:
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:461:30: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! SHARED_EXTERNAL_POINTER_TAGS(CHECK_SHARED_EXTERNAL_POINTER_TAGS)
npm ERR!                              ^
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:461:30: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:461:30: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:461:30: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:461:30: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:462:35: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! PER_ISOLATE_EXTERNAL_POINTER_TAGS(CHECK_NON_SHARED_EXTERNAL_POINTER_TAGS)
npm ERR!                                   ^
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:462:35: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:462:35: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:462:35: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:462:35: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:462:35: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:462:35: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:462:35: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:462:35: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:462:35: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:462:35: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:643:61: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR!     static_assert(kJSObjectType + 1 == kFirstJSApiObjectType);
npm ERR!                                                             ^
npm ERR!                                                             , ""
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:644:55: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR!     static_assert(kJSObjectType < kLastJSApiObjectType);
npm ERR!                                                       ^
npm ERR!                                                       , ""
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-internal.h:645:63: warning: 'static_assert' with no message is a C++17 extension [-Wc++17-extensions]
npm ERR!     static_assert(kFirstJSApiObjectType < kLastJSApiObjectType);
npm ERR!                                                               ^
npm ERR!                                                               , ""
npm ERR! In file included from ../lib/addon.cc:1:
npm ERR! In file included from ../lib/./wrapped_re2.h:4:
npm ERR! In file included from ../../nan/nan.h:60:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/node.h:73:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8.h:24:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-array-buffer.h:13:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-object.h:9:
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-maybe.h:106:45: error: no template named 'is_lvalue_reference_v' in namespace 'std'; did you mean 'is_lvalue_reference'?
npm ERR!   template <class U, std::enable_if_t<!std::is_lvalue_reference_v<U>>*>
npm ERR!                                        ~~~~~^~~~~~~~~~~~~~~~~~~~~
npm ERR!                                             is_lvalue_reference
npm ERR! /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/v1/type_traits:910:29: note: 'is_lvalue_reference' declared here
npm ERR! struct _LIBCPP_TEMPLATE_VIS is_lvalue_reference : _BoolConstant<__is_lvalue_reference(_Tp)> { };
npm ERR!                             ^
npm ERR! In file included from ../lib/addon.cc:1:
npm ERR! In file included from ../lib/./wrapped_re2.h:4:
npm ERR! In file included from ../../nan/nan.h:60:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/node.h:73:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8.h:24:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-array-buffer.h:13:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-object.h:9:
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-maybe.h:106:69: error: expected '(' for function-style cast or type construction
npm ERR!   template <class U, std::enable_if_t<!std::is_lvalue_reference_v<U>>*>
npm ERR!                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-maybe.h:123:43: error: no template named 'is_lvalue_reference_v' in namespace 'std'; did you mean 'is_lvalue_reference'?
npm ERR! template <class T, std::enable_if_t<!std::is_lvalue_reference_v<T>>* = nullptr>
npm ERR!                                      ~~~~~^~~~~~~~~~~~~~~~~~~~~
npm ERR!                                           is_lvalue_reference
npm ERR! /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/v1/type_traits:910:29: note: 'is_lvalue_reference' declared here
npm ERR! struct _LIBCPP_TEMPLATE_VIS is_lvalue_reference : _BoolConstant<__is_lvalue_reference(_Tp)> { };
npm ERR!                             ^
npm ERR! In file included from ../lib/addon.cc:1:
npm ERR! In file included from ../lib/./wrapped_re2.h:4:
npm ERR! In file included from ../../nan/nan.h:60:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/node.h:73:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8.h:24:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-array-buffer.h:13:
npm ERR! In file included from /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-object.h:9:
npm ERR! /Users/jonathanstegall/Library/Caches/node-gyp/19.1.0/include/node/v8-maybe.h:123:67: error: expected '(' for function-style cast or type construction
npm ERR! template <class T, std::enable_if_t<!std::is_lvalue_reference_v<T>>* = nullptr>
npm ERR!                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
npm ERR! 19 warnings and 4 errors generated.
npm ERR! make: *** [Release/obj.target/re2/lib/addon.o] Error 1
npm ERR! gyp ERR! build error 
npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
npm ERR! gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:201:23)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
npm ERR! gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:293:12)
npm ERR! gyp ERR! System Darwin 22.1.0
npm ERR! gyp ERR! command "/usr/local/Cellar/node/19.1.0/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
npm ERR! gyp ERR! cwd /usr/local/lib/node_modules/@automattic/vip/node_modules/re2
npm ERR! gyp ERR! node -v v19.1.0
npm ERR! gyp ERR! node-gyp -v v9.1.0
npm ERR! gyp ERR! not ok

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jonathanstegall/.npm/_logs/2022-11-16T16_16_16_397Z-debug-0.log
➜  minnpost-wordpress git:(master) ✗ 
```

## (Optional) Additional notes

- I do use Node with Homebrew. As mentioned, I uninstalled and reinstalled it. Node is `v19.1.0` and npm is `8.19.3`.
- My upgrade to Ventura is relatively recent. I did run `xcode-select --install` successfully.

**Comment by jonathanstegall** [comment]
Update: I've realized that I am able to successfully install older versions. Running `npm install -g @automattic/vip@2.22.0` completes without error, so this seems to be a new issue in 2.23.0.

**Comment by mjangda** [comment]
Looks like it might be something specific to Node 19. I've been able to reproduce there on macOS Monterey but not on Node 16 or 18.

So, alternate workaround for now is to switch to Node 18.

**Comment by rinatkhaziev** [comment]
I have a similar issue on node #14, it's not exactly the same but node-gyp is also involved.

**Comment by ibelong2u** [comment]
Confirming this on Windows 10 as well, with all versions of node.
I'm able to install the previous version using `npm install -g @automattic/vip@2.22.0`.

**Comment by lomeraniel** [comment]
The same is happening for vip version 2.24.0.

**Comment by spenserhale** [comment]
I experienced the same failure to build on node-gyp. Switching from Node 19 to Node 18 resolved the issue.

Maybe we should rename to issue to clarify that issue appears to be Node 19 and not a specific macOS version.

**Comment by katag9k** [comment]
Similar issue here but I'm still on Monterey

`node v18.13.0 (npm v8.19.3)`

```
npm install -g @automattic/vip

npm ERR! code 1
npm ERR! path /Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/@automattic/vip/node_modules/re2
npm ERR! command failed
npm ERR! command sh -c -- node-gyp rebuild
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.1.0
npm ERR! gyp info using node@18.13.0 | darwin | x64
npm ERR! gyp info find Python using Python version 3.9.10 found at "/usr/local/opt/python@3.9/bin/python3.9"
npm ERR! gyp info spawn /usr/local/opt/python@3.9/bin/python3.9
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/@automattic/vip/node_modules/re2/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/Users/kata/Library/Caches/node-gyp/18.13.0/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/Users/kata/Library/Caches/node-gyp/18.13.0',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/Users/kata/Library/Caches/node-gyp/18.13.0/<(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/@automattic/vip/node_modules/re2',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! No receipt for 'com.apple.pkg.CLTools_Executables' found at '/'.
npm ERR!
npm ERR! No receipt for 'com.apple.pkg.DeveloperToolsCLILeo' found at '/'.
npm ERR!
npm ERR! No receipt for 'com.apple.pkg.DeveloperToolsCLI' found at '/'.
npm ERR!
npm ERR! Traceback (most recent call last):
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/xcode_emulation.py", line 1502, in XcodeVersion
npm ERR!     version_list = GetStdoutQuiet(["xcodebuild", "-version"]).splitlines()
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/xcode_emulation.py", line 1563, in GetStdoutQuiet
npm ERR!     raise GypError("Error %d running %s" % (job.returncode, cmdlist[0]))
npm ERR! gyp.common.GypError: Error 1 running xcodebuild
npm ERR!
npm ERR! During handling of the above exception, another exception occurred:
npm ERR!
npm ERR! Traceback (most recent call last):
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py", line 45, in <module>
npm ERR!     sys.exit(gyp.script_main())
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 662, in script_main
npm ERR!     return main(sys.argv[1:])
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 654, in main
npm ERR!     return gyp_main(args)
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 639, in gyp_main
npm ERR!     generator.GenerateOutput(flat_list, targets, data, params)
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/generator/make.py", line 2455, in GenerateOutput
npm ERR!     writer.Write(
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/generator/make.py", line 830, in Write
npm ERR!     self.WriteSources(
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/generator/make.py", line 1305, in WriteSources
npm ERR!     cflags = self.xcode_settings.GetCflags(
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/xcode_emulation.py", line 662, in GetCflags
npm ERR!     archs = self.GetActiveArchs(self.configname)
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/xcode_emulation.py", line 512, in GetActiveArchs
npm ERR!     xcode_archs_default = GetXcodeArchsDefault()
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/xcode_emulation.py", line 119, in GetXcodeArchsDefault
npm ERR!     xcode_version, _ = XcodeVersion()
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/xcode_emulation.py", line 1513, in XcodeVersion
npm ERR!     version = CLTVersion()  # macOS Catalina returns 11.0.0.0.1.1567737322
npm ERR!   File "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/xcode_emulation.py", line 1551, in CLTVersion
npm ERR!     return re.search(regex, output).groupdict()["version"]
npm ERR! AttributeError: 'NoneType' object has no attribute 'groupdict'
npm ERR! gyp ERR! configure error
npm ERR! gyp ERR! stack Error: `gyp` failed with exit code: 1
npm ERR! gyp ERR! stack     at ChildProcess.onCpExit (/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:284:16)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
npm ERR! gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:291:12)
npm ERR! gyp ERR! System Darwin 21.6.0
npm ERR! gyp ERR! command "/Users/kata/.nvm/versions/node/v18.13.0/bin/node" "/Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
npm ERR! gyp ERR! cwd /Users/kata/.nvm/versions/node/v18.13.0/lib/node_modules/@automattic/vip/node_modules/re2
npm ERR! gyp ERR! node -v v18.13.0
npm ERR! gyp ERR! node-gyp -v v9.1.0
npm ERR! gyp ERR! not ok
```

**Comment by sjinks** [comment]
Fixed in 2.25.1.
